src/main/java/com/laserfiche/api/client/apiserver/TokenClient.java

```diff
@@ -3,16 +3,14 @@
 import com.laserfiche.api.client.model.CreateConnectionRequest;
 import com.laserfiche.api.client.model.SessionKeyInfo;
 
-import java.util.concurrent.CompletableFuture;
-
 public interface TokenClient extends AutoCloseable {
 
     /**
      * @param repoId Repository name
      * @param body   Request body that contains username, password and grant type
      * @return Create an access token successfully.
```

Comment:
I noticed this now. the description doesn't read appropriately.

Reply:
Yes, let me update the code.

Reply:
Thanks.